interface/themes/core/navmenu.scss

```diff
@@ -123,12 +123,12 @@ body {
     height: 110px;
 }
 
-/* $GLOBALS['logoBarHeight'] */
-
+/* $GLOBALS['logoBarHeight']
+removed as doesn't make sense and ruins responsive sjp
 .navbar {
     height: 22px;
 }
-
+*/
```

Comment:
delete, delete, delete!
:)

Reply:
yep, I know you don't like leaving patched out code in codebase. I get it but sometimes we need to leave a history behind with a comment why something is being removed. imo.
not so much here but I will leave so maybe @tywrenn can go through that css/scss and remove the dead overrides. He prob knows where those are better than me. If not i'll clean it up sometime later.

Reply:
i think i just like demolition :) definitely do whatever you think makes sense here.